model/src/main/kotlin/config/SnippetChoices.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.model.config
+
+import org.ossreviewtoolkit.model.RepositoryProvenance
+import org.ossreviewtoolkit.model.config.snippet.SnippetChoice
+
+/**
+ * A collection of snippet choices for a given provenance.
+ */
+data class SnippetChoices(
+    /**
+     * The provenance this snippet choice applies to.
+     */
+    val provenance: Provenance,
+
+    /**
+     * The snippet choices for this package.
+     */
+    val choices: List<SnippetChoice>
+)
+
+/**
+ * The URL of the [RepositoryProvenance] the snippet choice applies to.
+ */
+data class Provenance(val url: String)
```

Comment:
Can you give this class another name to avoid a name conflict if some code has to use this and the other `Provenance` class? I don't have a good idea, maybe `TargetProvenance`?

Reply:
Or also move to a `snippet` sub-package?

Reply:
I was meaning to only move.